content/de/blog/hacktoberfest-2021/index.md

```diff
@@ -0,0 +1,121 @@
+{
+    "title": "Hacktoberfest 2021: Macht mit!",
+    "type": "blog",
+    "date": "2021-09-19T11:00:0+02:00",
+    "description": "Wir halten 2021 auch wieder eine Hacktoberfest-Aktion ab. Mach' bei Datenanfragen.de mit und gewinne einen Webcam-Slider, Sticker und vielleicht ein T-Shirt!",
+    "featured_image": "hacktoberfest-2021",
+    "tags": [ "hacktoberfest", "digitalocean", "github", "open source", "pull request"],
+    "authors": [ "malte", "baltpeter" ],
+    "notices": "Titelfoto angepasst nach: „[maple leaves photo](https://unsplash.com/photos/Pu-en3ew8wY)“ von [Fey Marin](https://unsplash.com/@feymarin) ([Unsplash-Lizenz](https://unsplash.com/license))"
+}
+
+
+~~O'zapft is!~~ Der Oktober und damit auch das Hacktoberfest stehen wieder vor der Tür. Auch dieses Jahr wollen wir als Datenanfragen.de e.V. wieder am Hacktoberfest teilnehmen und freuen uns auf Eure Einreichungen für unser Projekt! Wie auch schon in den letzten Jahres gibt es auch wieder etwas zu gewinnen.
+
+{{< featuredImg alt="Vor einem Hintergrund aus Ahornblättern steht: \"Hacktoberfest 2021: Macht mit!\"" >}}
+
+Wie auch das [ursprüngliche Hacktoberfest](https://hacktoberfest.digitalocean.com/) von Digital Ocean, wollen wir mit unserer Aktion Menschen die Mithilfe an Open-Source-Projekten näher bringen und freuen uns über jeden Beitrag für Datenanfragen.de und unsere Schwesterseiten. Konkret heißt das für Dich: Wenn Du an unserer Aktion teilnimmst und etwas zu Datenanfragen.de beiträgst gewinnst Du einen praktischen Webcam-Slider und ein Sticker-Set (jeweils begrenzt auf die ersten 100 Teilnehmer_innen) und erhältst vielleicht auch ein T-Shirt von uns.
```

Comment:
```suggestion
Wie auch das [ursprüngliche Hacktoberfest](https://hacktoberfest.digitalocean.com/) von DigitalOcean, wollen wir mit unserer Aktion Menschen die Mithilfe an Open-Source-Projekten näher bringen und freuen uns über jeden Beitrag für Datenanfragen.de und unsere Schwesterseiten. Konkret heißt das für Dich: Wenn Du an unserer Aktion teilnimmst und etwas zu Datenanfragen.de beiträgst, gewinnst Du einen praktischen Webcam-Slider und ein Sticker-Set (jeweils begrenzt auf die ersten 100 Teilnehmer_innen) und erhältst vielleicht auch ein T-Shirt von uns.
```

Reply:
Imo "Schwesterseiten" makes it sound like they are projects run by different people.